presto-native-execution/presto_cpp/main/PrestoMain.cpp

```diff
@@ -30,8 +29,3 @@ int main(int argc, char* argv[]) {
   presto.run();
   PRESTO_SHUTDOWN_LOG(INFO) << "Exiting main()";
 }
-
-// Initialize singleton for the reporter.
-folly::Singleton<facebook::velox::BaseStatsReporter> reporter([]() {
```

Comment:
Dummy reporter is not really required. But I have moved it here:https://github.com/prestodb/presto/pull/22360/files#diff-42be14913ed8d3de507cebca449ab9255e5e0aa6b005d4c45091e50b65fd0febR59

Reply:
@karteekmurthys Lets keep it here instead of moving. The reasoning is there is a inherited class on top of PrestoMain  which registers Meta specific Stats reporting service. Now, since we moved it - the inherited class and new code path both try to register the singleton. Lets keep the dummy registration here.